drivers/source/SFDP.cpp

```diff
@@ -460,4 +462,4 @@ int sfdp_detect_addressability(uint8_t *bptbl_ptr, sfdp_bptbl_info &bptbl_info)
 #endif
 
 } /* namespace mbed */
-#endif /* (DEVICE_SPI || DEVICE_QSPI) */
+#endif /* (DEVICE_SPI || DEVICE_QSPI || DEVICE_OSPI) */
```

Comment:
Missing newline.

Reply:
Sorry, I do not  understand your meaning, could you give some more information?